src/Extend/View.php

```diff
@@ -30,11 +30,11 @@ class View implements ExtenderInterface, LifecycleInterface
      * You can also pass variables into a view: for more information, see https://laravel.com/api/8.x/Illuminate/View/Factory.html#method_make
      *
      * @param  string  $namespace: The name of the namespace.
-     * @param  string|array  $hints: This is a path (or an array of paths) to the folder(s)
+     * @param  string[]  $hints: This is a path (or an array of paths) to the folder(s)
```

Comment:
the typing no longer matches the description of the parameter.

If it really has to be an array, then the text can be updated to remove the "or an array" portion.

Likewise, the `array` type-hint could be added to the method. Unless this is for undocumented backward-compatibility?

Just like my other comment above, it seems like this was intended to be a simple string as well. Maybe `string|string[]` is the expected type-hint?

Reply:
Yup, I clearly was confused there.